routes/articles.js

```diff
@@ -10,4 +10,15 @@ router.get('/', (req, res) => {
     })
 })
 
+router.post('/article', (req,res) => {
+    const db =req.app.get('db')
+    const title = req.body.title
+    const body = req.body.body
+    const author = req.body.author
+    db.query('insert into articles(??,??,??)',[title,body,author], (err) => {
+        if(err) console.log('408')
```

Comment:
Its better to throw the error.  In case an error occurs, this will log `408` and still respond with `200`.

Throwing the error will return `500` _(internal server error)_ and log the error.

Reply:
Fixed